packages/visual-dev/src/components/Button/Button.tsx

```diff
@@ -59,7 +61,7 @@ const StyledButton = styled.button<
 >`
   ${Styles.universal_base}
   ${(props) => Styles[props.buttonType].base}
-  ${(props) => props.pill && Styles.pill}
+  /* ${(props) => props.pill && Styles.pill} */
```

Comment:
Flag comment

Reply:
Impact buttons don't have a pull style, I'll remove the commented out code